### DISCUSSION TITLE
RegexGenerator in latest alpine docker image not working

### DISCUSSION THREAD

**Discussion by Alumniminium** [dotnet/runtime#75151] [Q&A] [answered]
Not sure if the nightly container is just out of date, just discovered it doesn't build

error CS0246: The type or namespace name 'RegexGeneratorAttribute' could not be found
.net 7.0.100-rc.1.22415.4
MSBuild version 17.4.0-preview-22409-02+4ac5d65de

**Comment by stephentoub** [comment, marked as answer]
RegexGeneratorAttribute was renamed to GeneratedRegexAttribute
